<launch>

  <arg name="config_file" default="$(find svio)/../config/openloris_vio.yaml"/>

  <node pkg="svio" type="svio_syn_node" name="svio_syn_node" output="screen" args="$(arg config_file)">
    <param name="config_file" type="string" value="$(arg config_file)"/>
  </node>

  <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find svio)/../config/rviz_config_init.rviz"/>
  
</launch>